dataform/ecom-report-transform/README.md

```diff
@@ -0,0 +1,82 @@
+# `optimics/dataform-ecom-report-transform`
+
+# Ecom Report Transform
+
+## Overview
+
+This Dataform package is designed to help e-shops evaluate ecommerce data from
+Google Analaytics 4. It extracts information about ecommerce events and item parameters
+and stores it in a new dataset in BigQuery. The name of the dataset will be `ga4_datamart`.
```

Comment:
> The name of the dataset will be `ga4_datamart`.

This is confusing. The name of the target dataset is later said to be defined in the sqlx file, however, this statement implies, that the ecom-report-transform always puts data into `ga4_datamart`